api/src/download.py

```diff
@@ -0,0 +1,106 @@
+import MySQLdb
+import csv
+
+from flask import Blueprint
+from flask import request as req
+from flask import abort
+from flask import send_file
+
+from datetime import datetime
+
+from schema import schema_col_names
+from config import load_config
+
+download_bp = Blueprint('download', __name__)
+
+config = load_config()
+
+@download_bp.route('/')
+def download():
+
+    # START : same as ballots.py endpoint from here to "END"
```

Comment:
design-wise, I think it makes sense for download to be explicitly tied to the ballot endpoint (since the result of that endpoint is what we are downloading - so in the code, we also want this code to be dependent on the ballot endpoint code. 

e.g. if the ballot endpoint code were to change, then we would always have to remember to update it here, which is not good design choice imo. I think in `ballots.py` you can define a new function(s) to extrapolate out the shared code, and use those function(s) both in `ballots()` and here in `download()` by just importing them from `ballots.py` into this file.

Reply:
Fixed